mautic/mautic
# Theme panel headers

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | x
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description: Styling the layout of theme-selection panels so the buttons don't get pushed beneath their container

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Create/upload a theme with a long name
2. Go to the edit view of the email/page builder
3. Observe the panel buttons - buttons get pushed down

#### Steps to test this PR:
1. Create/upload a theme with a long name
2. Go to the edit view of the email/page builder
3. Observe the panel buttons - buttons remain in place
